auslib/test/admin/views/test_releases.py

```diff
@@ -417,6 +417,12 @@ def testDeleteWithoutPermissionForAction(self):
         ret = self._delete("/releases/b", username="bob", qs=dict(data_version=1))
         self.assertStatusCode(ret, 403)
 
+    def testDeleteWithProductAdminPermission(self):
+        ret = self._delete("/releases/a", username="billy", qs=dict(data_version=1))
+        self.assertStatusCode(ret, 200)
+        ret = self._delete("/releases/b", username="billy", qs=dict(data_version=1))
+        self.assertStatusCode(ret, 403)
```

Comment:
You've done well covering all of the important cases, but tests like this should be split in two - one for the positive case, one for the negative. Others below need to be split up as well.